editor/vim/coc-settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "languageserver": {
+        "jsonnet": {
+        "command": "jsonnet-language-server",
+        "args": ["-t"],
+        "rootPatterns": [".git/"],
+        "filetypes": ["jsonnet"]
```

Comment:
If I'm reading structure right, these should be indented a bit. Also, I think filetypes should include `libsonnet` as well.

Edit: I do see the filetypes here matches `jsonnet-language-server.vim` config. (cc @julienduchesne) Maybe we change this in that file too? I haven't messed around in vim/emacs with the lsp, so maybe it's doing something behind the scenes that I'm not privy to.

Reply:
+1 for adding `libsonnet` as filetype.

Reply:
thanks!, +1 to both, will fix